src/hooks/useMarkdownStyle.ts

```diff
@@ -0,0 +1,52 @@
+import type {MarkdownStyle} from '@expensify/react-native-live-markdown';
+import {useMemo} from 'react';
+import FontUtils from '@styles/utils/FontUtils';
+import variables from '@styles/variables';
+import useTheme from './useTheme';
+
+function useMarkdownStyle(): MarkdownStyle {
+    const theme = useTheme();
+
+    const markdownStyle = useMemo(
+        () => ({
+            syntax: {
+                color: theme.syntax,
+            },
+            link: {
+                color: theme.link,
+            },
+            h1: {
+                fontSize: variables.fontSizeh1,
+            },
+            blockquote: {
+                borderColor: theme.border,
+                borderWidth: 4,
+                marginLeft: 0,
+                paddingLeft: 6,
```

Comment:
Blockquote element has style of `inline-block` which is forcing cursor to be inside of this block and overlap with characters. We solved this by adding `padding-right` so that cursor has some space for itself

Reply:
FYI `paddingRight` is a no-op on mobile (Android & iOS), it works only on web.